.github/workflows/python-ci.yaml

```diff
@@ -0,0 +1,32 @@
+name: Python CI
+
+on:
+  push:
+    branches: [ master ]
```

Comment:
I am not sure if it is always master? Or this is something run daily(?) on master? thanks m.

Reply:
that means "on push to branch master"